### DISCUSSION TITLE
Playing around with alternate glyphs

### DISCUSSION THREAD

**Discussion by mgpjm** [coldtype/coldtype#95] [Q&A] [answered]
Hello,

I'm trying to playing with fonts which have multiple alternative glyphs.
I'm finding hard time to understand:
- how can I indicate which specific alternate I want for a specific glyphs?
- I'm allowed to choose a specific set of character (so I define the font, and also the set?)
- There is a way to let randomize the different alternative for a specific glyph on an animation?  So like: in the sentence "COLDTYPE" show me randomly 3 types of "C" "O" "L" etc... 

Thank you very much
m

**Comment by stenson** [comment, marked as answer]
Interesting question! I don't think there's a purely "direct" way of accessing alternate glyphs, but Coldtype does support the standard opentype stylistic set approach, via the `ss01=True`, `ss02=True` etc. keywords that you can pass to a `StSt` or a `Style` object.

Here's an example of an animation that toggles the `ss06` and `ss07` sets on and off (if you're on a Mac this should work w/ the built-in "SFCompactItalic" font, otherwise any font with stylistic sets should work)

```python
from coldtype import *

at = AsciiTimeline(3, 30, """
                            <
    [ss06      ]
           [ss07        ]
""")

@animation((1400, 540), timeline=at, bg=1, render_bg=1)
def scratch(f):
    return (StSt("alter-\nnates", "SFCompactItalic", 200
        , ss06=at.ki("ss06").on()
        , ss07=at.ki("ss07").on())
        .align(f.a.r))
```

https://user-images.githubusercontent.com/89770/159141028-9f80cff2-1209-4ab8-9b31-f4d8c34012df.mp4

Is that close to what you had in mind?

**Comment by mgpjm** [comment]
Wow Rob, yeah that was definitely the effect I was looking for. I already tried out, it works in a really cool way.
Many ideas cross my minds about it, especially on some random effect, but still in this way seems pretty what I need!
